schema/types/base/metaProperties.go

```diff
@@ -61,7 +61,10 @@ func (metaProperties metaProperties) MutateMetaProperty(metaProperty types.MetaP
 }
 
 func (metaProperties metaProperties) Get(id types.ID) types.Property {
-	return metaProperties.GetMetaProperty(id).RemoveData()
+	if metaProperty := metaProperties.GetMetaProperty(id); metaProperty != nil {
+		return metaProperty.RemoveData()
+	}
+	return nil
```

Comment:
I think this is where the issue #142 was arising and seems to be fixed.
can we return empty struct rather than nil?

Reply:
empty struct might create issues later down the line